index.d.ts

```diff
@@ -390,6 +399,27 @@ declare module '@lightningjs/blits' {
     routes?: Route[]
   }
 
+  export interface ApplicationConfig<P extends Props, S, M, C, W> extends ComponentConfig<P, S, M, C, W> {
+    /**
+     * Router configuration
```

Comment:
we should mention that router config take precedence over routes - I guess there's no way to specifiy in typescript that we should either use router or routes but not both? @jfboeve you any idea on this?

Reply:
hey sorry for the late response. got a snippet for you [here](https://www.typescriptlang.org/play/?#code/FASwdgLgpgTgZgQwMZQAQGED2BbADpsKSLMOEAc1QG9hgBfW8aeZNAJUwFdmSzKbUqABaZMAawDOALlQByEeImyANMEEwu0aagkQY4cgG0AuvUaRYiFBgJ9qa1Bu6wZszACMAVlCQRZDpy0ZXX0wI1MGYAgAT1w0AEFcXAAbECQECBACXgpUAF4bPAIiCBzKADJUAAoHAUFHTRdUDmcYMtV6hucJAH4ZQgA3WAcGAB9ah3VGmD7UQeHOwKhtEIMTEeAASlokAl1UBCSymUSUtIyssDL8+ynWmTrBBUlXZ6UO+qXtQ3kEZOTMLJTIIGJFgLswPtDrgygAmE5JVLpTLZWy5Ap1L4yH5CP4AlRyCAAd0BER2ewgByOaPIAGYEWdkZdrhiAo1vr9-oDlISSUCPksYA9JsJRC85G8VCKsagcXjSRs6KhgEA)